content/setup/support/.babylonFileFormat.md

```diff
@@ -763,6 +764,203 @@ Here is a simple example of .babylon file:
         "coneOuterGain": 0,
         "connectedMeshId": 0,
         "localDirectionToMesh": [1, 0, 0]
-   }]
+   }],
+"spriteManagers": [
+{
+	"name": "treesManager",
+	"capacity": 2000,
+	"cellWidth": 512,
+	"cellHeight": 1024,
+	"fogEnabled": true,
+	"blendMode": 2,
+	"disableDepthWrite": false,
+	"pixelPerfect": false,
+	"texture": {
+		"tags": null,
+		"url": "https://assets.babylonjs.com/textures/palm.png",
+		"uOffset": 0,
+		"vOffset": 0,
+		"uScale": 1,
+		"vScale": 1,
+		"uAng": 0,
+		"vAng": 0,
+		"wAng": 0,
+		"uRotationCenter": 0.5,
+		"vRotationCenter": 0.5,
+		"wRotationCenter": 0.5,
+		"homogeneousRotationInUVTransform": false,
+		"isBlocking": true,
+		"name": "https://assets.babylonjs.com/textures/palm.png",
+		"hasAlpha": false,
+		"getAlphaFromRGB": false,
+		"level": 1,
+		"coordinatesIndex": 0,
+		"optimizeUVAllocation": true,
+		"coordinatesMode": 0,
+		"wrapU": 0,
+		"wrapV": 0,
+		"wrapR": 1,
+		"anisotropicFilteringLevel": 4,
+		"isCube": false,
+		"is3D": false,
+		"is2DArray": false,
+		"gammaSpace": true,
+		"invertZ": false,
+		"lodLevelInAlpha": false,
+		"lodGenerationOffset": 0,
+		"lodGenerationScale": 0,
+		"linearSpecularLOD": false,
+		"isRenderTarget": false,
+		"animations": [],
+		"invertY": false,
+		"samplingMode": 3,
+		"_useSRGBBuffer": false,
+		"internalTextureLabel": "https://assets.babylonjs.com/textures/palm.png",
+		"noMipmap": true
+	},
+	"sprites": [
+		{
+			"name": "tree",
+			"position": [
+				-21.262812196810966,
+				0,
+				-15.254702234794953
+			],
+			"color": [
+				1,
+				1,
+				1,
+				1
+			],
+			"width": 1,
+			"height": 2,
+			"angle": 0,
+			"invertU": false,
+			"invertV": false,
+			"isPickable": false,
+			"isVisible": true,
+			"useAlphaForPicking": false,
+			"animationStarted": false,
+			"fromIndex": 0,
+			"toIndex": 0,
+			"loopAnimation": false,
+			"delay": 1
+		},
+		{
+			"name": "tree",
+			"position": [
+				-0.6418444578590048,
+				0,
+				24.069820121717868
+			],
+			"color": [
+				1,
+				1,
+				1,
+				1
+			],
+			"width": 1,
+			"height": 2,
+			"angle": 0,
+			"invertU": false,
+			"invertV": false,
+			"isPickable": false,
+			"isVisible": true,
+			"useAlphaForPicking": false,
+			"animationStarted": false,
+			"fromIndex": 0,
+			"toIndex": 0,
+			"loopAnimation": false,
+			"delay": 1
+		}
+	]
+]
 }
 ```
+## SpriteManagers
+A SpriteManager is defined by the following JSON:
+```javascript
+[
+{
+	"name": string,
+	"capacity": int,
+	"cellWidth": int,
+	"cellHeight": int,
+	"fogEnabled": true,
+	"blendMode": int,
+	"disableDepthWrite": boolean,
+	"pixelPerfect": boolean,
+	"texture": {

```

Comment:
I think you should just say `"texture": Texture` and link to the "Textures" section.